shared_model/validators/transaction_validator.hpp

```diff
@@ -118,7 +118,14 @@ namespace shared_model {
         addInvalidCommand(reason, "CreateRole");
 
         validator_.validateRoleId(reason, cr.roleName());
-        validator_.validateRolePermissions(reason, cr.rolePermissions());
+        for (auto i : static_cast<const shared_model::proto::CreateRole &>(cr)
```

Comment:
Why can't you use rolePermissions()? As far as I understood it returns set of Role permissions and you can iterate over it

Reply:
That may trigger out of bounds for bitset, if the proto is invalid